tracer/test/Datadog.Trace.Tests/DataStreamsMonitoring/DataStreamsManagerTests.cs

```diff
@@ -28,7 +28,7 @@ public void WhenDisabled_DoesNotInjectContext()
         var headers = new TestHeadersCollection();
         var context = new PathwayContext(new PathwayHash(123), 1234, 5678);
 
-        dsm.InjectPathwayContext(context, headers);
+        dsm.InjectPathwayContextAsBase64String(context, headers);
```

Comment:
Don't we want to test both `InjectPathwayContext` and `InjectPathwayContextAsBase64String`? Or, if `InjectPathwayContext` is not used anymore, should we remove it from `DataStreamsManager`?

Reply:
I think this question from @lucaspimentel still stands as I understand it.

If `InjectPathwayContext` isn't used (or isn't meant to be used) it should probably be removed.
If not we should probably re-add the tests for it and new ones for `InjectPathwayContextAsBase64String`